server/index.js

```diff
@@ -68,6 +68,9 @@ api.get('/test', findUserWithPhoneNumber, addSubscriptionToTable);
 
 app.use('/api', api);
 
+app.get('/api/test', findUserWithPhoneNumber, addSubscriptionToTable);
```

Comment:
I'm a little confused about this, isn't this something Tiffany added just to test things on her own repo? Do we need this in our master?